lesson_01/lesson_01.h

```diff
@@ -0,0 +1,3 @@
+﻿#pragma once
```

Comment:
Adding this header adds unnecessary compexity and no benefits)

Reply:
sure, thanx ) it was just auto generated. next time will check more carefully generated code.